quickwit-metastore/src/split_metadata_version.rs

```diff
@@ -138,6 +143,7 @@ pub(crate) struct SplitMetadataV1 {
 impl From<SplitMetadataV1> for SplitMetadata {
     fn from(v1: SplitMetadataV1) -> Self {
         let source_id = v1.source_id.unwrap_or_else(|| "unknown".to_string());
```

Comment:
This should have been `_unknown` for the `source_id`.

Reply:
Maybe `_quickwit__unknown` on both node_id ?

Reply:
Yes, sounds good.